aces/Loris
# [timepoint, instrument] pageclass obj null warning

## Brief summary of changes

Each page loading of timepoint list, instrument list and instrument will trigger a warning "PHP Notice:  Trying to get property 'page' of non-object in UserPageDecorationMiddleware.php on line 94", which is not pleasant to see when you are working with them every day. The problem comes from the Loris logic to process first, if not having gone through then to fail back to timepoint/instrument. I don't want to modify the logic though I assume that two times' processing might not be necessary, but it is too soon to say not. As a result, I just add a check to avoid the PHP warning.

#### Testing instructions (if applicable)

1. Load a timepoint list, instrument list and instrument page, then check web server log. The fix will avoid the warning.

#### Link(s) to related issue(s)

* Resolves #  (Reference the issue this fixes, if any.)
